src/cli/repl/commands/repl-query.ts

```diff
@@ -132,6 +132,10 @@ export function asciiSummaryOfQueryResult(formatter: OutputFormatter, totalInMs:
 			result.push(`Query: ${bold(query, formatter)} (${out['.meta'].timing.toFixed(0)}ms)`);
 			result.push(`   ╰ [Dataflow Graph](${graphToMermaidUrl(out.graph)})`);
 			continue;
+		} else if(query === 'id-map') {
+			const out = queryResults as QueryResults<'id-map'>['id-map'];
+			result.push(`Query: ${bold(query, formatter)} (${out['.meta'].timing.toFixed(0)}ms)`);
+			result.push(`   ╰ [Id List](${[...out.idMap.keys()].toString()})`);
```

Comment:
probably better to use an explicit join here
```suggestion
			result.push(`   ╰ Id List: {${[...out.idMap.keys()].map(String).join(', ')}}`);
```

if this works :D